portality/lib/formulaic.py

```diff
@@ -274,8 +274,7 @@ def list_fields_in_order(self):
                 if field.group_subfields():
                     for sf in field.group_subfields():
                         fieldlist.append(sf)
-                else:
-                    fieldlist.append(field)
+                fieldlist.append(field)
```

Comment:
if this is meant to be ordered, should this be __before__ the subfield is added?

```python
for field in fs.fields():
    fieldlist.append(field)
    if field.group_subfields():
        for sf in field.group_subfields():
            fieldlist.append(sf)

```

Reply:
Doesn't really matter in this case but I changed it to avoid any misinterpretations.